src/valhalla_export_edges.cc

```diff
@@ -223,15 +232,18 @@ int main(int argc, char *argv[]) {
          edge.e->trans_down() || edge.e->IsTransitLine()) //these 2 should never happen
         continue;
 
-      //get the opposing edge as well
-      edge_t opposing_edge = opposing(reader, tile, edge);
-      edge_set.set(tile_set.find(opposing_edge.i.Tile_Base())->second + opposing_edge.i.id());
-      ++set;
-
       //shortcuts arent real and maybe we dont want ferries
       if(edge.e->is_shortcut() || (!ferries && edge.e->use() == Use::kFerry))
         continue;
 
+      //get the opposing edge as well (ensure a valid edge is returned)
```

Comment:
Could go either way on this - generally the cost of getting an opposing edge can be high if it reaches into a neighboring tile. I can move this back if you think it makes sense. Probably doesn't make a lot of performance diff though we could check that later.